modules/apps/callbacks/ibc_middleware.go

```diff
@@ -99,7 +99,8 @@ func (im IBCMiddleware) SendPacket(
 		return 0, err
 	}
 
-	callbackData, err := types.GetSourceCallbackData(im.app, data, sourcePort, ctx.GasMeter().GasRemaining(), im.maxCallbackGas)
+	packet := channeltypes.NewPacket(data, seq, sourcePort, sourceChannel, "", "", timeoutHeight, timeoutTimestamp)
```

Comment:
maybe we could add an inline comment?

Reply:
not needed because we just use the source port/channel ID in `getCallbackData` right? Maybe worth adding a comment explicitly stating why it's safe to discard.